index.html

```diff
@@ -0,0 +1,180 @@
+                    <li class="element1">CANOPY</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">Back End Dev</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.</p>
+                <ul class="bottom-list">
+                    <li><button class="language-buttons" type="button">html</button></li>
+                    <li><button class="language-buttons" type="button">css</button></li>
+                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                </ul>
+                <button class="p-button" type="button">See project</button>
+            </div> 
+        </section>
+        <section class="about-me">
+            <h1 class="title1">About<br>Myself</h1>
+            <p class="description">I’m a software developer! 
+                I can help you build a product, 
+                feature or website. Look through 
+                some of my work and experience! 
+                If you like what you see and have 
+                a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <div class="connect">
+                <h4>LET'S CONNECT</h4>
+            </div>
+            <div class="social">
+                <ul>
+                    <li>
+                        <a href="https://www.facebook.com/emmanuel.cortes.944"><img src="twit.svg" alt="twitter"></a>
+                    </li>
+                    <li>
+                       <a href="www.linkedin.com/in/emmanuel-cortés-padilla-49098214"><img src="linkedin.svg" alt="linkedin"></a> 
+                    </li>
+                    <li>
+                        <a href="#"><img src="m.svg" alt="m"></a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/emmanuelcortes2"><img src="git.svg" alt="git"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="peaceandlove.svg" alt="peace"></a>
+                    </li>
+                </ul>
+            </div> 
+            <div>
+                <button class="p-button" type="button">Get my resume</button>
+            </div>
+            <nav class="menu">
+                <ul>
+                  <li><a href="#">Languages</a>
+                    <ul>
+                      <li class="js-btn"><a href="#">JavaScript</a></li>
+                      <li class="ht-btn"><a href="#">Html</a></li>
+                      <li class="css-btn"><a href="#">CSS</a></li>
+                    </ul>
+                  </li>
+                  <li><a href="#">Frameworks</a></li>
+                  <li><a href="#">Skills</a></li>
+                </ul>
+                </nav>
```

Comment:
- [ ] Kindly, try to Follow the Figma design, In your project for "Languages", "Frameworks","Skills" there is no arrow indication just like design template . You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
Your Project Look likes this : 
![Capture1](https://user-images.githubusercontent.com/93598189/186738587-63317905-611d-4db0-93dd-e18cb2cda0be.PNG)

The Figma design Look likes : 

![Capture](https://user-images.githubusercontent.com/93598189/186738661-6687ee45-7db5-468e-9236-97f7fb8392f4.PNG)

Reply:
Ok. Fixed the problem, added bottom border and dropdown icons